apis/core.oam.dev/v1alpha2/core_types.go

```diff
@@ -640,6 +640,24 @@ type ApplicationConfigurationList struct {
 	Items           []ApplicationConfiguration `json:"items"`
 }
 
+// +kubebuilder:object:root=true
+
+// An ResourceTracker represents a tracker for track cross namespace resources
+// +kubebuilder:resource:scope=Cluster,categories={crossplane,oam}
+type ResourceTracker struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+}
+
+// +kubebuilder:object:root=true
+
+// ResourceTrackerList contains a list of ResourceTracker
+type ResourceTrackerList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata,omitempty"`
+	Items           []ResourceTracker `json:"items"`
```

Comment:
Can we add a status and record what's binding on it

Reply:
Yes, it's really important